src/optimize.jl

```diff
@@ -68,15 +68,19 @@ function _run_minizinc(dest::Optimizer)
     if isempty(filename)
         filename = joinpath(dir, "model.mzn")
     end
-    output = joinpath(dir, "model.ozn")
     open(filename, "w") do io
         return write(io, dest.inner)
     end
+    output = joinpath(dir, "model.ozn")
+    _stdout = joinpath(dir, "_stdout.txt")
     _minizinc_exe() do exe
-        return run(`$(exe) --solver $(dest.solver) -o $(output) $(filename)`)
+        cmd = `$(exe) --solver $(dest.solver) -o $(output) $(filename)`
+        return run(pipeline(cmd, stdout = _stdout))
```

Comment:
for some reason the "UNSATISFIABLE" would sometimes get printed to stdout rather than to the output file (e.g. for test_moi_var_domain_infeasible_fzn but not for test_moi_infeasible_fzn)

Reply:
Ah. Maybe this is why I didn't add it at the time.